projects.go

```diff
@@ -94,9 +95,10 @@ type Project struct {
 	} `json:"shared_with_groups"`
 	Statistics        *ProjectStatistics `json:"statistics"`
 	Links             *Links             `json:"_links,omitempty"`
-	CIConfigPath      *string            `json:"ci_config_path"`
+	CIConfigPath      string             `json:"ci_config_path"`
```

Comment:
@svanharmelen Is this an intentional change as I don't think it is related to the "pending delete"

Reply:
It’s not related but it is intentional. Could make a dedicated PR for it, but thought to just add it when I noticed it...

Reply:
You don't have to put this in a dedicated PR. I just want to rule out any accidental mistake. ;-)

Reply:
Thanks for bringing it up 👍🏻